services/IoT/applications/libraries/nodejs.md

```diff
@@ -97,18 +97,18 @@ To connect to {{site.data.keyword.iot_short_notm}}, submit a *connect* request,
 	});
 ```
 
-After successfully connecting to the {{site.data.keyword.iot_short_notm}} service, the application client sends a *connect* event. So all the logic can be implemented inside this callback function.
+After successfully connecting to the {{site.data.keyword.iot_short_notm}} service, the application client sends a *connect* event, so all the logic can be implemented inside this callback function.
```

Comment:
"After successfully connecting to the {{site.data.keyword.iot_short_notm}} service, the application client sends a *connect* event, so all the logic can be implemented inside this callback function."
all logic sounds funny - recommend changing "all" to "any"